armi/operators/operator.py

```diff
@@ -665,7 +674,14 @@ def _checkTightCouplingConvergence(self, activeInterfaces: list):
                 self._convergenceSummary[key].append(coupler.eps)
 
         reportingUtils.writeTightCouplingConvergenceSummary(self._convergenceSummary)
-        return all(converged)
+        converged = all(converged)
+        if converged:
+            runLog.important(
```

Comment:
Just one comment on these `runLog`s. In a draft PR, I have suggested these `runLog`s. I think they'd work well here.  

https://github.com/terrapower/armi/pull/1172/commits/01a08248cd9693c2d890cd81c5a111a660a9edaa